app/Http/Controllers/MemberController.php

```diff
@@ -12,63 +12,70 @@
 
 class MemberController extends Controller
 {
-    public function listMember(Request $request)
+    public function index(Request $request)
     {
-        $listMember= Member::all()->toArray();
+        $listMember = Member::all() ->toArray();
         return $listMember ;
     }
 
-    public function deleteMember($id)
+    public function create()
     {
-        $member=Member::find($id);
-        $member ->delete();
-        return "xóa thành công" ;
     }
 
-    public function getCreateMember()
+    public function show($id)
     {
-        return view('test');
     }
 
-    public function getEditMember($id)
+    public function edit($id)
     {
-        $item = Member::find($id)->toArray();
-        return view('editmember', ['id'=>$id, 'item'=>$item]);
     }
 
-    public function editMember(StoreCreateMember $request)
+    public function destroy(Request $request)
     {
-        $data=$request->all();
-        $memberEdit=Member::find($data['id']);
-        $memberEdit->name=$data['name'];
-        $memberEdit->phone_number=$data['phone_number'];
-        $memberEdit->information=$data['information'];
-        $memberEdit->birthday=$data['birthday'];
-        $memberEdit->position_id=$data['position_id'];
-        $memberEdit->gender=$data['gender'];
-        if ($request->hasFile('avatar')) {
-            $file = $request->avatar;
-            $file->move("img", $file->getClientOriginalName());
-            $memberEdit->avatar = $file->getClientOriginalName();
+        $id = $request -> id;
+        if ($member = Member::find($id)) {
+        $listMember = Member::all() ->toArray();
+        return $listMember ;
+        }
+        return ["message" => "Doesn't Exit Item"];
+    }
+
+    public function update(StoreCreateMember $request)
+    {
+        $data = $request->all();
+        if ($memberEdit = Member::find($data['id'])) {
+            $memberEdit->name = $data['name'];
+            $memberEdit->phone_number = $data['phone_number'];
+            $memberEdit->information = $data['information'];
+            $memberEdit->birthday = $data['birthday'];
+            $memberEdit->position_id = $data['position_id'];
+            $memberEdit->gender = $data['gender'];
+            if ($request->hasFile('avatar')) {
+                $file = $request->avatar;
+                $file->move("img", $file->getClientOriginalName());
+                $memberEdit->avatar = mt_rand(1000, 10000).'-'.$file->getClientOriginalName();
+            }
+                $memberEdit->save();
```

Comment:
indentation wrong! Pls fix this.

Reply:
i don't understand this error?? becase indentation of if